tests/snappi/pfc/files/m2o_oversubscribe_lossless_lossy_helper.py

```diff
@@ -0,0 +1,402 @@
+# uncompyle6 version 3.9.0
+# Python bytecode version base 2.7 (62211)
+# Decompiled from: Python 3.10.4 (tags/v3.10.4:9d38120, Mar 23 2022, 23:13:41) [MSC v.1929 64 bit (AMD64)]
+# Embedded file name: /var/johnar/sonic-mgmt/tests/snappi/multi_dut_rdma/files/rdma_helper.py
+# Compiled at: 2023-02-10 09:15:26
```

Comment:
Is there a testplan for these testcases? If not, can you please add a testplan describing the test topology, testcase steps and expected result

Reply:
testplan is part of the PR https://github.com/sonic-net/sonic-mgmt/pull/5079